ibis/backends/duckdb/tests/conftest.py

```diff
@@ -2,13 +2,54 @@
 from pathlib import Path
 
 import ibis
+from ibis.backends.conftest import TEST_TABLES
 from ibis.backends.tests.base import BackendTest, RoundAwayFromZero
 
 
 class TestConf(BackendTest, RoundAwayFromZero):
     def __init__(self, data_directory: Path) -> None:
         self.connection = self.connect(data_directory)
 
+    @staticmethod
+    def load_data(data_dir, script_dir, **kwargs):
```

Comment:
```suggestion
    def load_data(data_dir, script_dir, database="ibis_testing", **kwargs):
```

Reply:
This is fine for now. I will address in a follow up.